modules/dhcp_common/free_ips.rb

```diff
@@ -29,11 +29,11 @@ def mark_ip_as_allocated(ip_address)
     def clean_up_allocated_ips
       @m.synchronize do
         loop do
-          logger.debug("Starting allocated ip addresses cleanup pass...")
+          logger.trace("Starting allocated ip addresses cleanup pass for DHCP ...")
```

Comment:
This looks like an unrelated change. It'd be better to separate that out to a different PR.